docs/api/app.md

```diff
@@ -725,10 +725,27 @@ Returns `string` - User operating system's locale two-letter [ISO 3166](https://
 
 ### `app.getSystemLocale()`
 
-Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, the `NSLocale` object is used instead.
+Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, `[NSLocale currentLocale]` is used instead. To get the user's current system language, which is not always the same as the locale, it is better to use `app.getPreferredSystemLanguages()` and take the first result.
+
+Note that the system locale is not always the most preferred system language:
+
+* On Windows 11, the locale returned is the one used for the regional format. For example, if the regional format is `zh-CN` and the most preferred system language is `fr-FR`, then `zh-CN` is returned.
+* On macOS Monterey, the locale is a string containing a mix of the most preferred system language and the region. For example, if the most preferred language is `pt-BR` but the region is Finland, the locale is `pt-FI` instead of `pt-BR`.
+
+Different operating systems also use the regional data differently:
+
+* Windows 11 uses the regional format to render dates and times.
+* macOS Monterey uses the region for formatting numbers, selecting the currency symbol to use, and rendering dates and times.
 
 **Note:** This API must be called after the `ready` event is emitted.
 
+### `app.getPreferredSystemLanguages()`
+
+Returns `string[]` - The user's preferred system languages from most preferred to least preferred, including the country codes if applicable. A user can modify and add to this list on Windows or macOS through the Language and Region settings.
+Some languages returned by this API appear in a different format than locales returned by `app.getLocale()`. For example, on Windows and macOS, the language code for Simplified Chinese starts with `zh-Hans`, and the language code for Traditional Chinese starts with `zh-Hant`.
```

Comment:
Why are we comparing with `app.getLocale` here ? The api only returns custom locale value supported by the runtime when generating the resource pak files https://source.chromium.org/chromium/chromium/src/+/main:build/config/locales.gni;l=25-110 which is always different from the values returned by both `app.getSystemLocale` and `app.getPreferredSystemLanguages` which are values returned by the OS. I feel like the comparison adds confusion.

```suggestion

```

Reply:
I think the comparison is helpful because if I didn't know better, I would have thought that the newer APIs returned values consistent with `app.getLocale()`
Maybe there's an alternative way I can emphasize that the APIs return values in different formats.